draft-ietf-tls-md5-sha1-deprecate-02.xml

```diff
@@ -157,7 +158,7 @@
      This document updates <xref
      target="RFC5246" /> and <xref target="RFC7525" />
      in such a way that MD5 and SHA-1 MUST NOT
-     be used for digital signatures.  However, this document does not deprecate SHA-1 in HMAC for record protection.
+     be used for digital signatures.  However, this document does not deprecate MD5 or SHA-1 in HMAC for record protection.
```

Comment:
I believe that it would be good to indicate HMAC-SHA1 and HMAC-MD5 are still considered secured at the time of publication of this document and indicate the reference to the document this is mentioned [RFC6151] [RFC6194]. 
This would clarify it has not been deprecated y another document.

Reply:
I am concerned that adding text that reaffirms that HMAC-MD5 & HAMC-SHA1 is still good is going to set up some weird reference problems. It seems like it could be argued that if this I-D re-affirms HMAC-MD5 & HAMC-SHA1 that we are updating 6151 and 6914; this would mean we would need to add updates 6151 and 6194 to the headers. But, we’re not really changing anything. Hoping this would work:

    However, this document does not deprecate MD5 or SHA-1 in HMAC
    for record protection; consult updates to [RFC6151] and
    [RFC6194]and their updates for the current state status.

And, we need to copy this text back to the security considerations as the same sentence appear there.

Reply:
this works for me.